articles/overview/what-is-quantum-computing.md

```diff
@@ -12,45 +12,51 @@ uid: microsoft.quantum.overview.what
 
-Quantum computing harnesses the unique behavior of quantum physics to provide a new and very powerful model of computing. The theory of quantum physics posits that matter, at a quantum level can be in a superposition of multiple classical states. And those many states interfere with each other like waves in a tide pool.
+Quantum computing harnesses the unique behavior of quantum physics to provide a new and very powerful model of computing. The theory of quantum physics posits that matter, at a quantum level can be in a superposition of multiple classical states. And those many states interfere with each other like waves in a tide pool.  The state of matter after a measurement is assumed to "collapse" into one of the classical states. Thereafter, repeating the same measurement will produce the same classical result.  Quantum entanglement occurs when particles interact in ways such that the quantum state of each cannot be completely described independently of the others, even if the particles are physically far apart.  
 
-Quantum computing stores information in quantum states of matter and uses quantum operations to compute on that information, by harnessing and learning to program quantum interference.
+Quantum computing stores information in quantum states of matter and use its quantum nature of superposition and entanglement to realize quantum operations that compute on that information, thereby harnessing and learning to program quantum interference.
 
 Quantum computing might sound daunting, but with the right resources you can start building quantum applications today.
 
 ## The qubit
 
-In quantum computing, a quantum bit - **qubit** - is a unit of quantum information, like a classical bit. Where classical bits hold a single binary value such as a 0 or 1, the state of a qubit can be in a **superposition of 0 and 1**.  Visually, A qubit can be thought of as a direction in space (also known as a vector).  Within that space, the two classical states are two directions in that space. A qubit can be in any of the possible directions.  
+Quantum computing defines computing concepts that reflect the quantum behavior.  Quantum computing begins with the notion of a qubit.  In quantum computing, a quantum bit - **qubit** - is a unit of quantum information, like a classical bit. Where classical bits hold a single binary value such as a 0 or 1, the state of a qubit can be in a **superposition** of 0 and 1 simultaneously.  
```

Comment:
I'd suggest avoiding phrasing that gets too close to the "two places at once" misunderstanding, as that's a very common misconception that readers may enter with.

```suggestion
Quantum computing defines computing concepts that reflect the quantum behavior.  Quantum computing begins with the notion of a qubit.  In quantum computing, a quantum bit - **qubit** - is a unit of quantum information, like a classical bit. Where classical bits hold a single binary value such as a 0 or 1, the state of a qubit can be in a **superposition** of 0 and 1.  
```

Reply:
wiki "qubit" defines the superposition using "simultaneously":  "However, quantum mechanics allows the qubit to be in a coherent superposition of both states/levels simultaneously, a property which is fundamental to quantum mechanics and quantum computing. "  And our conceptual docs:  "Perhaps the simplest is to see that quantum theory can be interpreted as saying that matter, at a quantum level, is simultaneously in a host of different possible configurations (known as states) at the same time. "

Reply:
I definitely recognize it's a common way to explain superposition, no disagreement. I do think, though, that as literally written, Wikipedia's summary is incorrect (as are other parts of our own conceptual documentation), and is incorrect in a way that has created confusion for our readers in the past. That said, I also do recognize that's my own $0.02, and is worth taking with a grain of salt.

Reply:
Let's leave it as is, consistent with our other doc for now.  We can discuss post Ignite.